include/phoenix/cffi/ModelScript.h

```diff
@@ -14,7 +14,168 @@ typedef phoenix::model_script PxModelScript;
 typedef struct PxInternal_ModelScript PxModelScript;
 #endif
 
-// TODO
+typedef enum : uint8_t {
+	af_none = 0,
+	af_move = 1,
+	af_rotate = 2,
+	af_queue = 4,
+	af_fly = 8,
+	af_idle = 16,   
+} PxAnimationFlags;
+
+typedef enum : uint8_t {
+	forward = 0,  ///< The animation samples are played from first to last.
+	backward = 1, ///< The animation samples are played from last to first.
+} PxAnimationDirection;
+
+typedef enum {
+	unknown,
+	create_item,
+	insert_item,
+	remove_item,
+	destroy_item,
+	place_item,
+	exchange_item,
+	fight_mode,
+	place_munition,
+	remove_munition,
+	draw_sound,
+	undraw_sound,
+	swap_mesh,
+	draw_torch,
+	inventory_torch,
+	drop_torch,
+	hit_limb,
+	hit_direction,
+	dam_multiply,
+	par_frame,
+	opt_frame,
+	hit_end,
+	window,
+} PxEventTagType;
+
+typedef enum {
+	fist,       ///< The player fights with his fists.
+	one_handed, ///< The player wields a one-handed weapon.
+	two_handed, ///< The player wields a two-handed weapon.
+	bow,        ///< The player wields a bow.
+	crossbow,   ///< The player wields a crossbow.
+	magic,      ///< The player casts a magic spell.
+	none,       ///< The player is not in a fighting stance.
+	invalid,    ///< A fight mode which acts as an `unset` marker. Added for OpenGothic compatibility.
+} PxEventFightMode;
```

Comment:
Enum members must be prefixed with `Px` too. Ideally the naming convention should be for example:

```c
typedef enum {
   PxAnimationDirectionForward = 0,
   PxAnimationDirectionBackward = 1,
} PxAnimationDirection;
```

Also, all enum members should have an assigned value. I forgot to add that on the phoenix side but I'll do that later. For now, just number them sequentially from 0, please.